python/scripts/do_release.py

```diff
@@ -108,6 +108,11 @@ def main(self):
             print("Current version is not newer. Exiting.")
             return
 
+        # copy README.md from top level to python folder so that appears on pypi
+        readme = ROOT / "README.md"
+        python_readme = ROOT / "python" / "README.md"
+        python_readme.write_text(readme.read_text())
```

Comment:
Worth adding README.md to the `python/` folders `.gitignore` so that it is not accidentally committed in the future?